torch/testing/_internal/distributed/distributed_test.py

```diff
@@ -1282,6 +1373,27 @@ def test_all_reduce_sum_cuda(self):
                 rank_to_GPU,
             )
 
+        @unittest.skipIf(
+            BACKEND != "gloo" and BACKEND != "nccl",
+            "Only Gloo backend will have CUDA allReduce tested",
```

Comment:
The test runs if backend is NCCL or Gloo, so this message needs to be updated right?